### PR TITLE
Update filterobjects help_settings (resolves #2269)

### LINKED ISSUE #2269: Main help text for FilterObjects missing the help info for "Relabel additional objects" option

### PR DESCRIPTION
**Pull request by bethac07**

### CODE REVIEW

**Comment by 0x00b1** [comment]
Nice. Squash and I'll merge. :)

**Comment by 0x00b1** [comment]
http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request

**Comment by bethac07** [comment]
Bookmarked.

**Comment by 0x00b1** [comment]
👍

File: cellprofiler/modules/filterobjects.py [hunk 1]
```diff
@@ -379,7 +378,8 @@ def help_settings(self):
                 self.rules_directory,
                 self.rules_file_name, self.rules_class,
                 self.enclosing_object_name,
-                self.wants_outlines, self.outlines_name]
+                self.wants_outlines, self.outlines_name,
+                self.additional_object_button]
```
Comment (mcquin) [inline cellprofiler/modules/filterobjects.py]: Unrelated change?

**Comment by bethac07** [comment]
Nope- it only goes to the help if you add it to help_settings

**Comment by 0x00b1** [comment]
@bethac07 please rebase
